giskard/testing/tests/data_quality.py

```diff
@@ -0,0 +1,64 @@
+"""
+Module for data quality tests.
+"""
+from giskard.ml_worker.testing.test_result import TestResult
+from giskard.ml_worker.testing.registry.decorators import test
+from giskard.datasets.base import Dataset
+
+@test(name="Data Uniqueness Test")
+def uniqueness_test(dataset: Dataset, column: str):
+    """
+    Test for checking the uniqueness of data in a column.
+
+    Args:
+        dataset (Dataset): The dataset to test.
+        column (str): The column to check for uniqueness.
+
+    Returns:
+        TestResult: The result of the test.
+    """
+    column_data = dataset.df[column]
+    uniqueness_ratio = len(column_data.unique()) / len(column_data)
+    return TestResult(passed=uniqueness_ratio == 1)
+
+@test(name="Data Completeness Test")
+def completeness_test(dataset: Dataset):
+    """
+    Test for checking the completeness of data in a dataset.
+
+    Args:
+        dataset (Dataset): The dataset to test.
+
+    Returns:
+        dict: A dictionary with the completeness score for each column.
+    """
+    completeness_scores = {}
+    for column in dataset.df.columns:
+        column_data = dataset.df[column]
+        completeness_ratio = len(column_data.dropna()) / len(column_data)
+        completeness_scores[column] = completeness_ratio
+    return completeness_scores
+
+@test(name="Data Range and Validity Test")
+def range_validity_test(dataset: Dataset, column: str, min_value=None, max_value=None, valid_values=None):
```

Comment:
In my opinion it would make more sense to split into two tests: 

- `range_validity_test(dataset: Dataset, column: str, min_value: Optional[float] = None, max_value: Optional[float] = None)`
- `allowed_values_validity_test(dataset: Dataset, valid_values: Sequence[Any])`.